src/main/java/net/smartcosmos/extension/tenant/DevKitUserManagementService.java

```diff
@@ -6,24 +6,19 @@
 import org.springframework.context.annotation.Import;
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 import org.springframework.security.crypto.password.PasswordEncoder;
-import org.springframework.web.servlet.config.annotation.EnableWebMvc;
-import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
 
 import net.smartcosmos.annotation.EnableSmartCosmosEvents;
 import net.smartcosmos.annotation.EnableSmartCosmosExtension;
-import net.smartcosmos.annotation.EnableSmartCosmosSecurity;
 import net.smartcosmos.extension.tenant.config.AnonymousAccessSecurityConfiguration;
 
 @EnableSmartCosmosExtension
 @EnableSmartCosmosEvents
-@EnableSmartCosmosSecurity
-@EnableWebMvc
-@Import({ AnonymousAccessSecurityConfiguration.class })
+//@EnableSmartCosmosSecurity
```

Comment:
Don't we lose OAuth authentication for the endpoints, if we remove `@EnableSmartCosmosSecurity`?